src/types/auth.ts

```diff
@@ -0,0 +1,17 @@
+/* Types for Auth related methods */
+
+export type authData = {
```

Comment:
```suggestion
export type AuthData = {
```
You can put these in a separate file but personally I tend to just keep them in the same file that they are most correlated to (ie function/class where they are used)

Reply:
@nbragdon Moved these in to auth.ts .